tests/conftest.py

```diff
@@ -11,6 +15,8 @@
     BaseViolation,
 )
 
+ALL_VIOLATIONS_TYPE: TypeAlias = Dict[ModuleType, List[BaseViolation]]
```

Comment:
`ALL_VIOLATIONS_TYPE` is a constant.
While `AllViolationsType` is a type :)

Please, rename it.